extensions/ql-vscode/src/helpers.ts

```diff
@@ -266,6 +269,36 @@ export function isFolderAlreadyInWorkspace(folderName: string) {
   );
 }
 
+/** Check if the current workspace is the CodeTour and open the workspace folder.
+ * Without this, we can't run the code tour correctly.
+ **/
+export async function prepareCodeTour(): Promise<void> {
+  if (workspace.workspaceFolders?.length) {
+    const currentFolder = workspace.workspaceFolders[0].uri.fsPath;
+
+    // We need this path to check that the file exists on windows
+    const tutorialWorkspacePath = join(
+      currentFolder,
+      "tutorial.code-workspace",
+    );
+    const toursFolderPath = join(currentFolder, ".tours");
+
+    if (
+      existsSync(tutorialWorkspacePath) &&
+      existsSync(toursFolderPath) &&
+      !isCodespacesTemplate()
```

Comment:
Minor: comment here why we need this negated condition. The explanation you put in the PR description is helpful.

Reply:
Added a comment.